src/api/index.test.js

```diff
@@ -342,6 +343,56 @@ it('createPipelineRun', () => {
   });
 });
 
+it('createPipelineRunAtProxy', () => {
+  const mockDateNow = jest
+    .spyOn(Date, 'now')
+    .mockImplementation(() => 'fake-timestamp');
+  const pipelineRef = 'fake-pipelineRef';
+  const resources = {};
+  const params = {};
+  const serviceAccount = 'fake-serviceAccount';
+  const timeout = 'fake-timeout';
+  const payload = {
+    namespace: 'fake-namespace',
+    pipelineRef: 'fake-pipelineRef',
```

Comment:
You have variables defined above for these so should use them to ensure consistency. I don't see `namespace` referenced anywhere else in this test, is it needed?

```diff
-pipelineRef: 'fake-pipelineRef',
+pipelineRef,
-resources: {},
+resources,
-params: {},
+params,
-serviceAccount: 'fake-serviceAccount',
+serviceAccount,
-timeout: 'fake-timeout',
+timeout
```

Reply:
Sorry, I forgot about that.